hedera-node/src/test/java/com/hedera/services/records/TransactionRecordServiceTest.java

```diff
@@ -84,7 +77,8 @@ class TransactionRecordServiceTest {
                             null,
                             EntityId.fromAddress(Address.BLS12_PAIRING),
                             55,
-                            0));
+                            0,
+                            CallOperationType.OP_CALL));
```

Comment:
Can we check why all the `EvmCreateTransactionWithSidecars` tests aren't failing?
I'd expect the logic that compares the expected actions to use the `OP_CREATE`

Reply:
These tests do not actually call into HederaTracer for creating the contract actions but rather gets a mocked list of precreated actions.  The test which actually exercise HaderaTracer for creating contract actions are in HederaTracerTest.